test/piezo.js

```diff
@@ -276,18 +276,32 @@ exports["Piezo"] = {
   },
 
   note: function(test) {
-    test.expect(4);
+    test.expect(9);
 
-    // note delegates to tone;
+    // note delegates to frequncy, which delegates to tone;
     var toneSpy = this.sandbox.spy(this.piezo, "tone");
+    var frequencySpy = this.sandbox.spy(this.piezo, "frequency");
 
     // accepts octave.
```

Comment:
Similarly to comments for the `note` attribute of the `note` method...maybe drop in definition of difference between `note` and `octave` here?